mlir/lib/Dialect/PDL/IR/PDL.cpp

```diff
@@ -148,18 +148,20 @@ static ParseResult parseOperationOpAttributes(
   Builder &builder = p.getBuilder();
   SmallVector<Attribute, 4> attrNames;
   if (succeeded(p.parseOptionalLBrace())) {
-    auto parseOperands = [&]() {
-      StringAttr nameAttr;
-      OpAsmParser::UnresolvedOperand operand;
-      if (p.parseAttribute(nameAttr) || p.parseEqual() ||
-          p.parseOperand(operand))
+    if (failed(p.parseOptionalRBrace())) {
+      auto parseOperands = [&]() {
+        StringAttr nameAttr;
+        OpAsmParser::UnresolvedOperand operand;
+        if (p.parseAttribute(nameAttr) || p.parseEqual() ||
+            p.parseOperand(operand))
+          return failure();
+        attrNames.push_back(nameAttr);
+        attrOperands.push_back(operand);
+        return success();
+      };
+      if (p.parseCommaSeparatedList(parseOperands) || p.parseRBrace())
         return failure();
-      attrNames.push_back(nameAttr);
-      attrOperands.push_back(operand);
-      return success();
-    };
-    if (p.parseCommaSeparatedList(parseOperands) || p.parseRBrace())
-      return failure();
+    }
   }
```

Comment:
The github diff is not very informative here. 
The existing parsing is simply surrounded by 
```
if (failed(p.parseOptionalRBrace())) {
//[...]
}
```
to enable parsing `{}` as empty operationOpAttributes.